npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -408,10 +416,6 @@ def write_lock_files
           end
         end
 
-        def prepared_yarn_lockfile_content(content)
```

Comment:
Any idea what the purpose of these removals was before, and why it's OK to get rid of this?

Reply:
yeah pretty sure it was just a copy from the sub version resolver where it's used to update the sub-dependency